OBAKit/OBAApplication.m

```diff
@@ -93,6 +93,7 @@ - (void)registerAppDefaults {
     defaults[kUngroupedBookmarksOpenKey] = @(YES);
     defaults[OBAOptInToTrackingDefaultsKey] = @(YES);
     defaults[OBAAllowReviewPromptsDefaultsKey] = @(YES);
+    defaults[OBAMapSelectedTypeDefaultsKey] = MKMapTypeStandard;
```

Comment:
I'm guessing that this works only because `MKMapTypeStandard` is equal to `0`, which the compiler is interpreting as nil. Please box this. i.e. `@(MKMapTypeStandard)`